b0nbon/stack-overflow-lite
# Ft-162607948-user-login-and-signup

*What does this PR do*  
Adds sign up and login page

*Description of task to be completed*  
Create a login and signup page with the following features:-

1. login form
2. sign up form

*How should this be manually tested*

1. Clone this repository
2. Cd into the Stack-OverFlow-Lite folder
3. Open login.html in your browser

*What are the relevant pivotal tracker stories*  
#162607948 A User should be able to see the create an account and log in

*Screenshot*  
![login](https://user-images.githubusercontent.com/45699928/50038598-03e23800-0033-11e9-8f94-894bcb576680.JPG)
![signup](https://user-images.githubusercontent.com/45699928/50038599-03e23800-0033-11e9-8a68-233aa72d22df.JPG)
